### DISCUSSION TITLE
Version selection for ceph-csi

### DISCUSSION THREAD

**Discussion by liumingyuan2233** [ceph/ceph-csi#3767] [Q&A] [answered]
My ceph cluster is 15.2.17 and the kubenets cluster is 1.20.0. How do I choose the corresponding supported ceph csi version?

**Comment by riya-singhal31** [comment]
You can find the same over support matrix here, https://github.com/ceph/ceph-csi/blob/devel/README.md

**Comment by lmysky** [comment]
> 您可以在这里找到相同的过度支持矩阵，https://github.com/ceph/ceph-csi/blob/devel/README.md

Thank you. May I ask if the CEPH CSI version is only applicable to version 3.7.0 or higher? Because I saw that the ceph csi before version 3.7.0 was abandoned.

**Comment by Rakshith-R** [comment, marked as answer]
> > 您可以在这里找到相同的过度支持矩阵，https://github.com/ceph/ceph-csi/blob/devel/README.md
> 
> Thank you. May I ask if the CEPH CSI version is only applicable to version 3.7.0 or higher? Because I saw that the ceph csi before version 3.7.0 was abandoned.

Please refer to https://github.com/ceph/ceph-csi/blob/devel/README.md#known-to-work-co-platforms .

```
NOTE:

The supported window of Ceph CSI versions is "N.(x-1)": (N (Latest major release) . (x (Latest minor release) - 1)).

For example, if the Ceph CSI latest major version is 3.8.0 today, support is provided for the versions above 3.7.0. If users are running an unsupported Ceph CSI version, they will be asked to upgrade when requesting support.
```

All CephCSI versions will still work. We will only be backporting fixes and handling issues with most recent releases as stated above due to limited bandwidth. Everyone is encouraged to upgrade and use the latest versions according to the given matrix.
